MicrosoftFluentUI.podspec

```diff
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   
   s.osx.deployment_target = "10.13"
```

Comment:
you need to change the podspec version so that when you update the git tag it include your changes.